client/src/app/components/task-list/task-list/task-list.component.html

```diff
@@ -3,3 +3,44 @@
     Add Task
   </button>
 </div>
+
+<div class="container" style="padding: 10px">
+  <div class="row">
+    <div *ngFor="let task of tasks" class="col-sm-3">
+      <div
+        class="card mb-3"
+        [ngClass]="{
+          'bg-light': task.status == 'To Do',
+          'bg-warning': task.status == 'In Progress',
+          'bg-danger': task.status == 'Stuck',
+          'bg-success': task.status == 'Done',
```

Comment:
nit: in JS, you should almost ALWAYS use `===` and `!==` instead of `==` and `!=`. `==` is more dangerous because it can convert types. (See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness# if you really want to know more)